l10n_it_fatturapa_out/tests/test_fatturapa_xml_validation.py

```diff
@@ -0,0 +1,676 @@
+                        },
+                    ),
+                    (
+                        0,
+                        0,
+                        {
+                            "account_id": self.a_sale.id,
+                            "product_id": self.product_order_01.id,
+                            "name": "Zed+ Antivirus",
+                            "quantity": 1,
+                            "product_uom_id": self.product_uom_unit.id,
+                            "price_unit": 4,
+                            "tax_ids": [(6, 0, {self.tax_22.id})],
+                        },
+                    ),
+                ],
+            }
+        )
+        self.assertTrue(invoice.fiscal_document_type_id)
+        invoice._post()
+        res = self.run_wizard(invoice.id)
+        attachment = self.attach_model.browse(res["res_id"])
+        self.set_e_invoice_file_id(attachment, "IT06363391001_00006.xml")
+
+        xml_content = base64.decodebytes(attachment.datas)
+        self.check_content(xml_content, "IT06363391001_00006.xml")
+
+    def test_7_xml_export(self):
+        self.product_product_10.default_code = False
+        self.product_order_01.barcode = False
+        self.company.partner_id.vat = "CHE-114.993.395 IVA"
+        self.company.partner_id.name = "Azienda estera"
+        self.company.partner_id.city = "Lugano"
+        self.company.partner_id.state_id = False
+        self.company.partner_id.country_id = self.env.ref("base.ch").id
+        self.company.fatturapa_tax_representative = self.intermediario.id
+        self.company.fatturapa_stabile_organizzazione = self.stabile_organizzazione.id
+        self.set_sequences(14, "2018-01-07")
+        invoice = self.invoice_model.create(
+            {
+                "name": "INV/2018/0014",
+                "invoice_date": "2018-01-07",
+                "partner_id": self.res_partner_fatturapa_2.id,
+                "journal_id": self.sales_journal.id,
+                # "account_id": self.a_recv.id,
+                "invoice_payment_term_id": self.account_payment_term.id,
+                "user_id": self.user_demo.id,
+                "move_type": "out_invoice",
+                "currency_id": self.EUR.id,
+                "invoice_line_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "account_id": self.a_sale.id,
+                            "product_id": self.product_product_10.id,
+                            "name": "Mouse Optical",
+                            "quantity": 1,
+                            "product_uom_id": self.product_uom_unit.id,
+                            "price_unit": 10,
+                            "tax_ids": [(6, 0, {self.tax_22.id})],
+                        },
+                    ),
+                ],
+            }
+        )
+        self.assertTrue(invoice.fiscal_document_type_id)
```

Comment:
ho provato ad eseguire i test in locale ma falliscono perchè non viene trovato il campo **fiscal_document_type_id** in **account.move**. Manca la dipendenza da **l10n_it_fiscal_document_type** nel manifest

Reply:
Ho eliminato le assert. Era l'unica cosa che creava la dipendenza.